fuel/app/tasks/robots.php

```diff
@@ -68,28 +66,28 @@ public static function run($speech = null)
 			      |___|   |___|
 			      /   \\   /   \\
 			     |_____| |_____|
-			     |HHHHH| |HHHHH|", 'blue');
-	}
+			     |HHHHH| |HHHHH|', 'blue');
+    }
 
-	/**
-	 * An example method that is here just to show the various uses of tasks.
-	 *
-	 * Usage (from command line):
-	 *
-	 * php oil r robots:protect
-	 *
-	 * @return string
-	 */
-	public static function protect()
-	{
-		$eye = \Cli::color("*", 'green');
+    /**
+     * An example method that is here just to show the various uses of tasks.
+     *
+     * Usage (from command line):
+     *
+     * php oil r robots:protect
+     *
+     * @return string
+     */
+    public static function protect()
+    {
+        $eye = \Cli::color('*', 'green');
 
-		return \Cli::color("
-					\"PROTECT ALL HUMANS\"
+        return \Cli::color('
+					"PROTECT ALL HUMANS"
 			          _____     /
-			         /_____\\", 'blue')."\n"
-.\Cli::color("			    ____[\\", 'blue').$eye.\Cli::color('---', 'blue').$eye.\Cli::color('/]____', 'blue')."\n"
-.\Cli::color("			   /\\ #\\ \\_____/ /# /\\
+			         /_____\\', 'blue')."\n"
+.\Cli::color('			    ____[\\', 'blue').$eye.\Cli::color('---', 'blue').$eye.\Cli::color('/]____', 'blue')."\n"
```

Comment:
__[phpmd]__
* Avoid using static access to class '\Cli' in method 'protect'.
* Avoid using static access to class '\Cli' in method 'protect'.
* Avoid using static access to class '\Cli' in method 'protect'.